src/components/send/SendContainer.js

```diff
@@ -98,12 +106,19 @@ export function SendContainer({ match, location }) {
     }
 
     const handleConfirm = () => {
+        Mixpanel.track("SEND Click submit button")
         setConfirm(true)
     }
 
     const handleSend = async () => {
-        await dispatch(sendMoney(id, parseNearAmount(amount)))
-        await dispatch(getBalance())
+        try {
```

Comment:
Looks like this pattern is used so frequently that will benefit from being extracted into function, so that looks more like this:

```js
await withTracking('SEND money`, async () => {
    await dispatch(sendMoney(id, parseNearAmount(amount)))
    await dispatch(getBalance()) 
});
```

basically have something like:

```js
    async function withTracking(name, fn) {
         try {
             Mixpanel.track(`${name} start`)
             await fn();
             Mixpanel.track(`${name} finish`)
         } catch (e) {
             Mixpanel.track(`${name} fail`, {error: e.message})
         }
    }
```

Reply:
will change this